packages/client/vite.config.ts

```diff
@@ -90,7 +91,9 @@ export default defineConfig(({ mode }) => {
     },
     resolve: {
       alias: {
-        crypto: 'crypto-js'
+        crypto: 'crypto-js',
+        '@opencrvs/commons/build/dist/authentication':
```

Comment:
Sorry, couldn't get both the TS compiler and vitest to agree with each other without this